chapters/annotations.tex

```diff
@@ -1263,11 +1281,11 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
   $\ldots$
   flow Modelica.Units.SI.Force f[3]
   annotation(unassignedMessage =
-      "All Forces cannot be uniquely calculated. The reason could be that the
+    "All Forces cannot be uniquely calculated. The reason could be that the
      mechanism contains a planar loop or that joints constrain the same motion.
```

Comment:
I'm not quite sure what you mean, but what I see is a pre-formatted message with lots of white-space after the line breaks that is going to give a really ugly result when included in the error message presented to the user.

I am generally in favor of the idea to try to set an example of good style in the specification that users can copy, although it may require some tricks to at the same time comply with the page width constrains of the specification.  In this case, I'll make use of string concatenation to fit the width constraint, although I might have considered using a very long line in an actual Modelica file.

Reply:
OK now?

Reply:
Sorry, I was not clear. The sentence "The reason could be..." is on two lines here. I was wondering if it was necessary to create that line break in the sentence manually because it is part of lsllisting. Sorry, I don't remember how the formatting of lines is performed inside lstlisting.

Reply:
We want all listings to have hard line breaks so that every single line fits within the width constraint of the PDF build.  This also improves readability of the HTML build, as one never needs to scroll horizontally to see the full contents of a listing.  Usually, this means one just has to combine suitable places for line breaks with proper indentation of the following lines, but it is different inside string literals as a line break isn't just whitespace there.  Instead, the line length constraint needs to be fulfilled by breaking up a long string in smaller parts, and concatenate the parts using the `+` operator.  This is what I did here, but as explained above, the reader needs to understand that this is probably not the way to do it in a real Modelica library.